README.md

```diff
@@ -91,9 +91,9 @@ function makeWobblyPoint(x, y, wobble) {
 
 The pattern above has more of the flexibility associated with traits or mixins. Each layer is expressed separately. The layers are then combined by a distinct maker function `makeWobblyPoint`. Different making functions can combine overlapping sets of layers in different manners.
 
-The code for expressing the combinable layers is written in the peculiar manner shown above, using generators. We introduce this peculiar generator pattern to solve a hard problem: The code in each layer needs to capture a lexical variable that refers to the object as a whole. However, the object-as-a-whole is not yet assembled, and will be assembled only by a distinct piece of code written in a distinct scope.
+The code for expressing the combinable layers is written in the peculiar manner shown above, using intermediate function. We introduce this peculiar pattern to solve a hard problem: The code in each layer needs to capture a lexical variable that refers to the object as a whole. However, the object-as-a-whole is not yet assembled, and will be assembled only by a distinct piece of code written in a distinct scope.
```

Comment:
using *an* intermediate

The object-as-a-whole explanation is no longer compelling because we're not assembling the object before binding it. There is no such hard problem.